karton/dashboard/app.py

```diff
@@ -202,17 +221,45 @@ def get_queues_api():
     )
 
 
+@app.route("/<queue_name>/restart", methods=["POST"])
+def restart_queue_tasks(queue_name):
+    state = KartonState(karton.backend)
+    queue = state.queues.get(queue_name)
+    if not queue:
```

Comment:
what do you think about a helper for this? I.e. some kind of `get_queue(state)` that either returns a queue or :abort:s with 404